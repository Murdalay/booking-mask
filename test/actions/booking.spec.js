import {
  changePassengersNumber,
  changeFrom,
  changeTo,
  changeFlightDates,
  submit,
  unsetInvalid,
  togglePassengersDropdown,
} from 'actions/booking';

describe('App', () => {
  it('changePassengersNumber should return an action', () => {
    expect(changePassengersNumber()).toMatchSnapshot();
  });

  it('changeFrom should return an action', () => {
    expect(changeFrom()).toMatchSnapshot();
  });

  it('changeTo should return an action', () => {
    expect(changeTo()).toMatchSnapshot();
  });

  it('changeFlightDates should return an action', () => {
    expect(changeFlightDates()).toMatchSnapshot();
  });

  it('submit should return an action', () => {
    expect(submit()).toMatchSnapshot();
  });

  it('unsetInvalid should return an action', () => {
    expect(unsetInvalid()).toMatchSnapshot();
  });

  it('togglePassengersDropdown should return an action', () => {
    expect(togglePassengersDropdown()).toMatchSnapshot();
  });
});
