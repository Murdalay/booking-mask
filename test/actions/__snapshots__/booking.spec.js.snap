// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App changeFlightDates should return an action 1`] = `
Object {
  "type": "BOOKING_CHANGE_FLIGHT_DATES",
}
`;

exports[`App changeFrom should return an action 1`] = `
Object {
  "payload": Object {
    "code": undefined,
  },
  "type": "BOOKING_CHANGE_FROM",
}
`;

exports[`App changePassengersNumber should return an action 1`] = `
Object {
  "payload": Object {
    "data": undefined,
  },
  "type": "BOOKING_CHANGE_PASSENGERS_NUMBER",
}
`;

exports[`App changeTo should return an action 1`] = `
Object {
  "payload": Object {
    "code": undefined,
  },
  "type": "BOOKING_CHANGE_TO",
}
`;

exports[`App submit should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "BOOKING_SUBMIT",
}
`;

exports[`App togglePassengersDropdown should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "BOOKING_TOGGLE_PASSENGERS_DROPDOWN",
}
`;

exports[`App unsetInvalid should return an action 1`] = `
Object {
  "payload": Object {},
  "type": "BOOKING_UNSET_INVALID",
}
`;
